build/package/nfpm.yaml

```diff
@@ -11,7 +11,7 @@ maintainer: "Kong Inc. <support@konghq.com>"
 description: |
   Kong is a distributed gateway for APIs and Microservices, focused on high performance and reliability.
 vendor: "Kong Inc."
-license: "ASL 2.0"
+license: "Apache-2.0"
```

Comment:
grep reveals:

```
.requirements:KONG_LICENSE="ASL 2.0"
build/package/nfpm.yaml:license: "ASL 2.0"
```

Any reason why KONG_LICENSE was not also included as well?

Reply:
Good catch.